guides/getting-started.md

```diff
@@ -45,7 +45,20 @@ export class PizzaPartyAppModule { }
 
 ## Step 3: Import the component modules
 
-Import the NgModule for each component you want to use: 
+Import MaterialModule in your NgModule:
+
+```ts
+import {MaterialModule} from '@angular/material';
+
+@NgModule({
+  ...
+  imports: [MaterialModule],
```

Comment:
Using `MaterialModule` is no longer valid. The `MaterialModule` has been deprecated because kitchen-sink modules are not treeshakable.